docs/sdk/tutorials/basic_project_setup.md

```diff
@@ -0,0 +1,5 @@
+# How to setup a basic Kili project
+
+In this tutorial, we will learn how to set up a basic Kili project.
+
+To access this notebook tutorial, click here: [Basic Project Setup](https://github.com/kili-technology/kili-python-sdk/blob/master/recipes/basic_project_setup.ipynb)
```

Comment:
I would simply put: 
```
In this tutorial, we will learn how to set up a basic Kili project: [Basic Project Setup Notebook](https://github.com/kili-technology/kili-python-sdk/blob/master/recipes/basic_project_setup.ipynb)
```

Reply:
How about:

To learn how to set up a basic Kili project, open this tutorial: [Basic Project Setup Notebook](https://github.com/kili-technology/kili-python-sdk/blob/master/recipes/basic_project_setup.ipynb).